### PR TITLE
Database to xml

### PR DESCRIPTION
**Pull request by AlexDeweert**
**NOTICE ME SENPAI**

### CODE REVIEW

File: StockTrader/src/main/resources/schema.sql [hunk 1]
```diff
@@ -33,15 +33,19 @@ CREATE TABLE investment (
...
 CREATE TABLE account_transaction_log (
-    transaction_num SERIAL PRIMARY KEY,
+    id SERIAL PRIMARY KEY,
     action varchar(255),
     funds integer,
     timestamp bigint,
-    username varchar(255)
+    username varchar(255),
+    server varchar(255),
+    "transactionNum" integer
```
Comment (Celwood93) [inline StockTrader/src/main/resources/schema.sql]: how come this has quotes?

File: StockTrader/src/main/resources/schema.sql [hunk 2]
```diff
@@ -75,13 +79,13 @@ BEGIN
         RETURN NULL;
       END IF;
 --       INSERT INTO account_transaction_log (action, funds, timestamp, username)
-      INSERT INTO account_transaction_log (action, funds, timestamp, username)
-      VALUES (action, funds, trunc(extract(epoch from now()) * 1000), NEW.user_id);
-      WITH temp (action,funds,timestamp,username) AS (values (action, funds, trunc(extract(epoch from now()) * 1000), NEW.user_id))
+      INSERT INTO account_transaction_log (action, funds, timestamp, username, server, "transactionNum")
+      VALUES (action, funds, trunc(extract(epoch from now()) * 1000), NEW.user_id, NEW.last_server, NEW.last_transaction_number);
```
Comment (Celwood93) [inline StockTrader/src/main/resources/schema.sql]: same here, whats with the quotes

File: StockTrader/src/main/resources/schema.sql (hunk 1 above)
Comment (AlexDeweert) [reply StockTrader/src/main/resources/schema.sql]: Postgres is an annoying bitch when it comes to capitalizing column names. The xml tag needs that to be camel case. Xmlforest function turns the column name into the tag name. The profs xml validated needs that tag to be camel case

File: StockTrader/src/main/resources/schema.sql (hunk 2 above)
Comment (AlexDeweert) [reply StockTrader/src/main/resources/schema.sql]: Same as above. To enforce capitalizing the column name. Postgres wants everything to be lower case.

**Pull request merged**